ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -134,3 +134,10 @@ sunbird_portal_preview_cdn_url:
 cdn_file_path:
 sunbird_portal_cdn_blob_url:
 desktop_app_storage_url: "https://{{sunbird_offline_azure_storage_account}}.blob.core.windows.net/{{offline_installer_container_name}}"
+
+
+# Druid-Proxy-API
+druid_proxy_replicas: 1
+druid_proxy_reservation_memory: 300M
+druid_proxy_limit_memory: 300M
+sunbird_druid_broker_host: "http://{{groups['broker'][0]}}"
```

Comment:
this group is not defined in Core inventory of swarm.

Reply:
move this variable to ansible/inventory/dev/Core/all.yaml
add host to private_repo/ansible/inventory/dev/Core/hosts : for adopters